packages/react-ui/src/widgets/RangeWidgetUI.js

```diff
@@ -133,40 +144,51 @@ function RangeWidgetUI({ data, min, max, limits, onSelectedRangeChange }) {
     const newValues = Object.assign([], inputsValues, { [index]: value }).sort(
       (a, b) => a - b
     );
-    setInputsValues(newValues);
-    setSliderValues([...newValues]);
-    onSelectedRangeChange && onSelectedRangeChange(newValues);
+    changeSliderValues(newValues);
+  };
+
+  const resetSlider = () => {
+    changeSliderValues([min, max]);
```

Comment:
Ok, that's a nice simplification.
But I think it should really remove the filter in this case. Here it will modify the filter to use min/max values. This will select all features, like if no filter were applied, but with the added cost of having to apply the useless filter to all features.

Look at how other widgets are coded: for example in [https://github.com/CartoDB/carto-react/blob/8f0b65e2d167e7e9748d7ec6ac03bf9b0f986b78/packages/react-widgets/src/widgets/CategoryWidget.js#L92-L97](CategoryWidget), when we clear the selected categories, we actually dispatch a `removeFilter` action.
Same for `BarWidget`, `HistogramWidget`, etc.

We should do the same here, modifying `onSelectedRangeChange` in `RangeWidget` to really remove the filter when the user uses the clear button.

Reply:
After Slack conversation, it appears I overlooked that RangeWidget can be initialized with its self min/max which are already a default filter that shouldn't be removed. So, the cases to remove the filter are more complex than what I thought, and just updating it will work in every cases, so :+1: with this solution.
We can always come back to it for optimization later if needed.